packages/commons-test/src/setupTestContainersVitestGlobal.ts

```diff
@@ -68,6 +60,7 @@ export function setupTestContainersVitestGlobal() {
     let startedMongodbContainer: StartedTestContainer | undefined;
     let startedMinioContainer: StartedTestContainer | undefined;
     let startedMailpitContainer: StartedTestContainer | undefined;
+    let startedElasticMQContainer: StartedTestContainer | undefined;
```

Comment:
considering [this](https://github.com/pagopa/interop-be-monorepo/pull/634/files#diff-66d8ac03423ff2deaccbba0f6bbb345006c243f3aa669944d2c76312ca75fbf9R10) comment, should the elastic MQ container be in commons-test?